src/create_helpers.ts

```diff
@@ -149,27 +169,30 @@ export function createFunction(doclet: IFunctionDoclet): ts.FunctionDeclaration
     if (doclet.name.startsWith('exports.'))
         doclet.name = doclet.name.replace('exports.', '');
 
-    return handleComment(doclet, ts.createFunctionDeclaration(
-        undefined,      // decorators
-        mods,           // modifiers
-        undefined,      // asteriskToken
-        doclet.name,    // name
-        typeParams,     // typeParameters
-        params,         // parameters
-        type,           // type
-        undefined       // body
-    ));
+    return handleComment(
+        doclet,
+        ts.createFunctionDeclaration(
+            undefined, // decorators
+            mods, // modifiers
+            undefined, // asteriskToken
+            doclet.name, // name
+            typeParams, // typeParameters
+            params, // parameters
+            type, // type
+            undefined, // body
+        ),
+    );
 }
 
-export function createClassMethod(doclet: IFunctionDoclet): ts.MethodDeclaration
-{
+export function createClassMethod(
+    doclet: IFunctionDoclet,
+): ts.MethodDeclaration {
     const mods: ts.Modifier[] = [];
     const params = createFunctionParams(doclet);
     const type = createFunctionReturnType(doclet);
     const typeParams = resolveTypeParameters(doclet);
 
-    if (!doclet.memberof)
-        mods.push(declareModifier);
+    if (!doclet.memberof) mods.push(declareModifier);
```

Comment:
control flow statements should always be followed by a newline.

Reply:
Same as above, would need to add in braces.